.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "dex-contracts"]
 	path = dex-contracts
-	url = https://github.com/gnosis/dex-contracts
```

Comment:
Was there something wrong with using https? Could this be the problem with the cloning?

Reply:
I wanted to push my dex-contracts changes upstream from the submodule and for that I had to get ssh activated. But it looks like travis doesn't like it so I will revert.